README.md

```diff
@@ -150,3 +150,117 @@ Currently in Doppler [we are always using the v1](https://github.com/MakingSense
   </body>
 </html>
 ```
+
+## File storage
+
+### AWS S3 storage configuration
+
+Unlayer have file storage support built-in with Amazon S3 that can be configured without any coding. Follow these steps to set up your S3 file storage.
+
+- Create S3 Bucket: The first step is to create a bucket in your AWS S3 console. You need to uncheck "'Block all public access" for S3 storage.
+
+  ![AWS S3 edit block puplic access](./docs/aws-uncheck-s3-block-all-public-access.png)
+
+- Create IAM User: Go to your AWS IAM console and create a new user with `Programmatic Access` so you can get the access key id and secret access key.
+
+- Attach Policy and Permissions: Aattach a minimum set of permissions, use the following policy.
+Make sure to replace BUCKET-NAME, ID-COUNT, ID-USER with your actual bucket settings.
+
+  ```json
+  {
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+              "Sid": "AllowFullAccessBucket",
+              "Effect": "Allow",
+              "Principal": {
+                  "AWS": "arn:aws:iam::ID-COUNT:user/ID-USER"
+              },
+              "Action": [
+                  "s3:GetBucketPublicAccessBlock",
+                  "s3:PutObject",
+                  "s3:GetObject",
+                  "s3:PutBucketPublicAccessBlock",
+                  "s3:GetBucketCORS",
+                  "s3:ListBucket",
+                  "s3:PutBucketCORS",
+                  "s3:GetBucketAcl",
+                  "s3:GetBucketLocation",
+                  "s3:PutObjectAcl",
+                  "s3:PutBucketTagging"
+              ],
+              "Resource": [
+                  "arn:aws:s3:::BUCKET-NAME/*",
+                  "arn:aws:s3:::BUCKET-NAME"
+              ]
+          }
+      ]
+  }
+  ```
+
+- Enable CORS for your S3 Bucket: Your S3 Bucket must have CORS (cross-origin resource sharing) enabled. Follow these steps:
+
+  - Go to your S3 Bucket in AWS Console
+
+  - Click the Permissions tab
+
+  - Click the CORS Configuration button
+
+  - Add the following in the configuration editor
+
+  ```json
+  [
+      {
+          "AllowedHeaders": [],
+          "AllowedMethods": [
+              "PUT",
+              "POST",
+              "DELETE",
+              "HEAD"
+          ],
+          "AllowedOrigins": [
+              "https://*.unlayer.com"
+          ],
+          "ExposeHeaders": []
+      }
+  ]
+  ```
+
+- Add Credentials in your Unlayer Project: Simply enter your information of S3 storage against your project. For this:
+
+  - Login to Unlayer dashboard
+
+  - Select your Project
+
+  - Click Settings in the navigation bar and select the Storage tab
+
+  - Click the Add Storage button
+
+  - Enter your Access Key Id, Secret Access Key, and Bucket
+
+  - Check Primary Storage at the end
+
+  ![AWS S3 edit block puplic access](./docs/config-unlayer-settings-storage-aws-s3.png)
```

Comment:
```suggestion
    ![AWS S3 edit block puplic access](./docs/config-unlayer-settings-storage-aws-s3.png)
```

(It is subordinated to the last item _"Check Primary Storage at the end"_, right?

Reply:
No, it is general image of all checklists.